.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Let us know about a bug!
+title: ''
+labels: new
```

Comment:
Good idea! I think I'll leave that off for now, because we use the `new` label as an indicator that someone on the team hasn't looked at the issue yet to triage, and if there are other labels along with `new` it might get lost.